ocp_utilities/monitoring.py

```diff
@@ -48,7 +46,8 @@ def __init__(
             resource_name (str): Prometheus API resource name
             client (DynamicClient): Admin client resource
             verify_ssl (bool): Perform SSL verification on query
-            bearer_token (str): Used for query OAuth with API endpoint
+            bearer_token (str): Used for query OAuth with API endpoint, this needs to be created via oc create token command
+                example: oc create token <resource_name> -n <namespace> --duration=<number>s
```

Comment:
Can we give here a more specific example?
Maybe a command to find on which ns is Prometheus and how to generate the token.
 `oc create token prometheus-token -n <namespace> --duration=<number>s`

Reply:
Sure. But it basically depends on the resource_name and namespace. I will add a sample that we use in cnv.